### PR TITLE
build: drop Node.js v10

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
This is a metadata update

**Did you add tests for your changes?**
N/A

**If relevant, did you update the documentation?**
N/A

**Summary**
Node.js `v10` reached EOL.

**Does this PR introduce a breaking change?**
Drops support for Node.js `v10`.

**Other information**
N/A

### CODE REVIEW

**Comment by alexander-akait** [comment]
Too early for us, we will do it late, we have other more priority tasks right now in roadmap

**Comment by jamesgeorge007** [comment]
Yup, candidate for the next major.

**Comment by alexander-akait** [comment]
Drop Node.js v10, drop webpack v4 support, removed hacks from our code for webpack v4, and couple breaking changes (I can provide them)

**Comment by alexander-akait** [comment]
Create https://github.com/webpack/webpack-cli/tree/next

**Review by evenstensberg — APPROVED** [review]

**Review by evenstensberg — COMMENTED** [review]
Nvm, following Alexander's advice

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "url": "https://github.com/webpack/webpack-cli.git"
   },
   "engines": {
-    "node": ">=10.13.0"
+    "node": ">=12.13.0"
```
Comment (alexander-akait) [inline package.json:11]: Do we need change it for other packages?

File: packages/webpack-cli/package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
   },
   "main": "./lib/index.js",
   "engines": {
-    "node": ">=10.13.0"
+    "node": ">=12.13.0"
```
Comment (alexander-akait) [inline packages/webpack-cli/package.json:16]: @jamesgeorge007 Let's add `engines.node` to all our packages to avoid future problems :+1:

**Comment by alexander-akait** [comment]
Done here https://github.com/webpack/webpack-cli/pull/2879

**Pull request closed without merging**